server/lib/auth.ts

```diff
@@ -51,6 +51,7 @@ export async function getDiscordUserAndGuilds(authCode: string) {
 export async function upsertUser(user: DiscordOauth2.User) {
   const result = await User.upsert({
     discord_user_id: user.id,
+    discord_name: user.username,
```

Comment:
So this just saves the username, but doesn't save the discriminator.  When I console.log the user here and log in, I see:
```
{                                              
  id: '437048770237169666',                    
  username: 'chimten',                         
  display_name: null,                          
  avatar: '450a0b622a7c905c4c75e190cb21f2df',  
  avatar_decoration: null,                     
  discriminator: '3819',                       
```

So we want to construct the `discord_name` in the format of `username#discriminator`